### ISSUE TITLE
Font size sometimes increasing after rendering

### ISSUE THREAD

**Issue by shanuvshaji** [mathjax/MathJax#1901]
in android MathView, sometimes font size increasing after rendering the equation.
![prb](https://user-images.githubusercontent.com/22024073/35147317-295b5fc4-fd34-11e7-9a6b-388f3f6e1471.jpg)

**Comment by dpvc** [comment]
There is not enough information here to address your issue.  

The only guess I can make is that often mobile devices will rescale the page in a way that causes MathJax output to be incorrectly sized.  See [this post](https://groups.google.com/forum/#!topic/mathjax-users/v0hYt3pvf_8) for some details.  One solution is to use a `<meta>` tag such as 

```
<meta name="viewport" content="width=device-width, initial-scale=1">
```

to prevent that.  Alternatively, you might try configuring MathJax to check for fonts changes using the [MatchWebFonts](http://docs.mathjax.org/en/latest/options/other/MatchWebFonts.html#configure-matchwebfonts) extension.  This will readjust the font size once the page's size has been determined (which is after MathJax runs initially).

perhaps one of these will work for you.  Otherwise, you will have to give more specific information about your setup.  Live examples are best.

**Comment by shanuvshaji** [comment]
how can I do that?  
this is what exactly what I did now,


 <io.github.kexanie.library.MathView
        android:id="@+id/math_view"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_gravity="center_vertical"
        android:padding="5dp"
        android:scrollbars="none"
        app:engine="MathJax" />



 MathView mMathView = (MathView) findViewById(R.id.math_view);
 mMathView.setText(".............................");
